changes/1822.misc.rst

```diff
@@ -0,0 +1,7 @@
+Docs - update discussion and social media links.
```

Comment:
The changenote in this file should be a 1 line description, suitable for inclusion in release notes. It generally shouldn't discuss specifics of the change. 
```suggestion
Social media and discussion links were updated.
```